octobot_commons/databases/bases/database_cache.py

```diff
@@ -0,0 +1,98 @@
+#  Drakkar-Software OctoBot-Commons
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_commons.errors as errors
+
+
+class DatabaseCache:
+    MAX_CACHE_SIZE = 512
+
+    def __init__(self):
+        self.rows_cache = {}
```

Comment:
I think you should have a look at https://www.geeksforgeeks.org/python-functools-lru_cache/ instead of having mutliple cache dict with a `MAX_CACHE_SIZE`

Reply:
already discussed, resolving this